plugin-utilities-test/src/main/kotlin/org/jetbrains/research/pluginUtilities/util/FileTestUtil.kt

```diff
@@ -0,0 +1,73 @@
+package org.jetbrains.research.pluginUtilities.util
+
+import com.intellij.psi.PsiFile
+import com.intellij.testFramework.fixtures.CodeInsightTestFixture
+import java.io.File
+
+enum class Type {
+    Input, Output
+}
+
+class TestFileFormat(private val prefix: String, private val extension: Extension, val type: Type) {
```

Comment:
This assumes that all tested operations work with input and output files (not directories). In my case, preprocessing is done for the entire project, it has an input directory and output directory. This abstraction cannot be used in my case if I understand everything correctly.

Reply:
Yes, thank you! I will fix it today to add an able of using folders as input